trigger-statements.txt

```diff
@@ -9,3 +9,16 @@ Users of multiple linters
 -------------------------
 
 coala provides a common command-line interface for linting and fixing all your code, regardless of the programming languages you use.
+
+
+General purpose trigger statement
+---------------------------------
+
+coala provides a common multiplatform interface for linting and fixing all
+your code, regardless of the programming language you use. Its convenient user
+interface eliminates a lot of common tasks for the developer. coala also
+facilitates people entering the world of open source by providing them
+valuable feedback on coding standards and formatting. So, in a nutshell coala
```

Comment:
'provides ... feedback..'
I think 'feedback' is the wrong word here, as it is a warm word, suggesting an intelligent agent.
coala is currently a cold heartless and very low intelligence feedback.

I think the intended point is coala simplifies the path for new OSS developers by giving them painless access to verification tools used by most OSS projects.